specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -52,7 +52,7 @@ These settings apply only when `--tag=package-2019-05` is specified on the comma
 
 ``` yaml $(tag) == 'package-2019-05'
 input-file:
-- Microsoft.RecoveryServices/stable/2019-05-13/bms.json
+- Microsoft.RecoveryServices/stable/2019-06-15/bms.json
```

Comment:
Please keep the old entry (2019-05-13) in-place

Reply:
Retained the old entry and created a new package (2019-06) which contains all the APIs.